### PR TITLE
Add check for omim gene entries used as xrefs

### PR DESCRIPTION
**Pull request by twhetzel**
Replaces https://github.com/monarch-initiative/mondo/pull/7984 since too many changes in `master` at this point.

See https://github.com/monarch-initiative/mondo/issues/7983 for context. Should also pass now since https://github.com/monarch-initiative/mondo/pull/8011.

### CODE REVIEW

**Comment by twhetzel** [comment]
Nico or Sabrina - can you review and merge this once approved?

**Review by sabrinatoro — APPROVED** [review]
looks good to me, but I am not the best person to identify possible errors in sparql queries. 
@matentzn please confirm this is ok. Thanks!

**Comment by twhetzel** [comment]
It's the same SPARQL query as the original PR. It's a new PR since there were too many changes in `master` in the time to sort out what needed to be updated in `mondo-edit.obo` to make this test pass.

**Pull request merged**
